config/imsim-config.yaml

```diff
@@ -291,5 +285,5 @@ output:
             dec: "$sky_pos.dec.deg"
             x: "$image_pos.x"
             y: "$image_pos.y"
-            flux: "$(@current_prof).flux"
+            flux: "$(@current_prof).calculateFlux(bandpass)"
```

Comment:
We probably don't want this I suspect.  This does a non-trivial calculation for every object, so it's going to slow things down a lot.  Probably better to just remove the flux column in the output.  Not sure if there is something appropriate we can replace it with that would just be a look-up, not a calculation.

Reply:
The revised code _is_ noticeably slower than before.  I think we can just omit it.

Reply:
That may also be because you aren't doing the fast thing for the faint objects.  (Depending on the input catalog how many faint objects you are using.)